ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/flow-details/gio-ps-flow-details.component.html

```diff
@@ -15,9 +15,17 @@
     limitations under the License.
 
 -->
+<ng-container *ngIf="flows.length; else emptyFlows">
+  <div class="header">
+    <div class="mat-h4">Flow details</div>
+  </div>
 
-<div class="header">
-  <div class="mat-h4">Flow details</div>
-</div>
+  <div class="content">🚧 Work in progress 🚧</div>
+</ng-container>
 
-<div class="content">🚧 Work in progress 🚧</div>
+<ng-template #emptyFlows>
+  <div class="emptyFlows">
+    <h2>No flows yet</h2>
+    <p class="mat-body-1">Flows allow you to customise the behaviour of your API event phases through configurable policies</p>
```

Comment:
British spelling of behaviour. We use US spelling in other parts of the site, but I don't know what has been decided upon 🤷‍♀️

Reply:
🏓 @audrey-tlm 😁 
I've totally copy and past it ? any suggestion @jourdiw ?

Reply:
```suggestion
    <p class="mat-body-1">Flows allow you to customize the behavior of your API event phases through configurable policies</p>
```